experimental/adaptive-dialog/csharp_dotnetcore/calendar-bot-cui/Dialogs/AcceptCalendarEntry/AcceptCalendarEntry.cs

```diff
@@ -0,0 +1,104 @@
+                    },
+                    new IfCondition()
+                    {
+                        Condition = "user.focusedMeeting == null",
+                        Steps = new List<IDialog>(){
+                            new SendActivity("You cannot accept any meetings because your calendar is empty"),
+                            new EndDialog()
+                        }
+                    },
+                    new SendActivity("[detailedEntryTemplate(user.focusedMeeting)]"),
+                    new ConfirmInput()
+                    {
+                        Property = "turn.AcceptCalendarEntry_ConfirmChoice",
+                        Prompt = new ActivityTemplate("Are you sure you want to accept this event?"),
+                        InvalidPrompt = new ActivityTemplate("Please Say Yes/No."),
+                    },
+                    new IfCondition()
+                    {
+                        Condition = "turn.AcceptCalendarEntry_ConfirmChoice",
+                        Steps = new List<IDialog>()
+                        {
+                            new SendActivity("{user.focusedMeeting.id}"),
+                            new IfCondition() // we cannot accept a entry if we are the origanizer
+                            {
+                                Condition = "user.focusedMeeting.isOrganizer != true",
+                                Steps = new List<IDialog>()
+                                {
+                                    new HttpRequest()
+                                    {
+                                        Property = "user.acceptResponse",
+                                        Method = HttpRequest.HttpMethod.POST,
+                                        Url = "https://graph.microsoft.com/v1.0/me/events/{user.focusedMeeting.id}/accept",
+                                        //Url = "https://graph.microsoft.com/v1.0/me/events/AAMkADY2MzM5M2UzLWQ0NmItNDU2My1hN2NjLTliMjRiYWE5YWQ4ZABGAAAAAADRv-cRMwIfQKntE9IXL-ciBwDVXUsVK2tOTK5RjTff3j-IAAAAAAENAADVXUsVK2tOTK5RjTff3j-IAAAfRHvTAAA=/accept",
+                                        Headers =  new Dictionary<string, string>()
+                                        {
+                                            ["Authorization"] = "Bearer {user.token.Token}",
+                                        },
+                                        Body = JObject.Parse(@"{
+                                          '1': '1'
+                                        }")
+                                      },
+                                    new SendActivity("Simple read back"),
+                                    //new SendActivity("[AcceptReadBack]")
+                                },
+                                ElseSteps = new List<IDialog>(){
+                                    new SendActivity("Your request can't be completed. You can't respond to this meeting because you're the meeting organizer.")
```

Comment:
>("Your request can't be completed. You can't respond to this meeting because you're the meeting organize [](start = 52, length = 104)

Change all hard coded activity to LG.